package.json

```diff
@@ -16,6 +16,7 @@
     "build": "lerna run build",
     "preci": "lerna run build",
     "ci": "npm-run-all format:validate ci:subpackages legal:*",
```

Comment:
this is how the entire repo should be CI-tested, so I would suggest something like `"ci": "npm run lint && npm run ts-typecheck && npm run test",` (or just the test?) **inside the package.json of your new package**.

Reply:
ummm... the ts-typecheck actually gives me quite some errors, but it looks like those are the expected ones. So exclude the testcases from being handled by typescript, if that works - or just remove this "ts-typecheck" script: there is not much actual TS code to check except for the model itself and that one is thoroughly tested by your tests.